.github/workflows/build.yml

```
@@ -114,62 +114,11 @@ jobs:
       with:
         go-version: '1.21'
 
-    - name: Build pgroll (Linux amd64)
-      run: CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-X github.com/xataio/pgroll/cmd.Version=${PGROLL_VERSION}" -o pgroll.linux.amd64
-
-    - name: Build pgroll (Linux arm64)
-      run: CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -ldflags "-X github.com/xataio/pgroll/cmd.Version=${PGROLL_VERSION}" -o pgroll.linux.arm64
-
-    - name: Build pgroll (MacOS amd64)
-      run: CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build -ldflags "-X github.com/xataio/pgroll/cmd.Version=${PGROLL_VERSION}" -o pgroll.macos.amd64
-
-    - name: Build pgroll (MacOS arm64)
-      run: CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 go build -ldflags "-X github.com/xataio/pgroll/cmd.Version=${PGROLL_VERSION}" -o pgroll.macos.arm64
-
-    - name: Build pgroll (Windows)
-      run: CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -ldflags "-X github.com/xataio/pgroll/cmd.Version=${PGROLL_VERSION}" -o pgroll.win.amd64
-
-    - uses: actions/upload-artifact@v3
-      with:
-        name: pgroll.linux.amd64
-        path: pgroll.linux.amd64
-
-    - uses: actions/upload-artifact@v3
-      with:
-        name: pgroll.linux.arm64
-        path: pgroll.linux.arm64
-
-    - uses: actions/upload-artifact@v3
-      with:
-        name: pgroll.macos.amd64
-        path: pgroll.macos.amd64
-
-    - uses: actions/upload-artifact@v3
-      with:
-        name: pgroll.macos.arm64
-        path: pgroll.macos.arm64
-
-    - uses: actions/upload-artifact@v3
+    - name: Run GoReleaser
+      uses: goreleaser/goreleaser-action@v3
       with:
-        name: pgroll.win.amd64
-        path: pgroll.win.amd64
-
-  release:
-    runs-on: ubuntu-latest
-    needs: [build]
-    if: startsWith(github.ref, 'refs/tags/')
-    steps:
-    - uses: actions/download-artifact@v3
-      with:
-        path: artifacts
-
-    - name: Release
-      uses: softprops/action-gh-release@v1
-      with:
-        fail_on_unmatched_files: true
-        files: |
-          artifacts/pgroll.linux.amd64/pgroll.linux.amd64
-          artifacts/pgroll.linux.arm64/pgroll.linux.arm64
-          artifacts/pgroll.macos.amd64/pgroll.macos.amd64
-          artifacts/pgroll.macos.arm64/pgroll.macos.arm64
-          artifacts/pgroll.win.amd64/pgroll.win.amd64
+        distribution: goreleaser
+        version: latest
+        args: release --clean
+      env:
+        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
```

This is already in the repository as `GIT_TOKEN`:

```suggestion
        GITHUB_TOKEN: ${{ secrets.GIT_TOKEN }}
```